roles/extract_openshift_installer/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+arch: x86_64
+version_filter: "[?(openshift_version == '{{ openshift_version }}') && (cpu_architecture == '{{ arch }}')]"
+release_image: "{{ (assisted_installer_release_images | json_query(version_filter))[0].url }}"
+extact_dest_path: /tmp/wip/extract/
```

Comment:
I think this is a typo? `s/extact/extract/` ? Could be easily mistyped in the inventory file

Reply:
Fixed